src/renderer/components/settings/Information.tsx

```diff
@@ -5,25 +5,29 @@
 // that can be found in the LICENSE file exposed on Github (readium) in the project repository.
 // ==LICENSE-END==
 
+import * as commonmark from "commonmark";
 import * as React from "react";
-
 import { connect } from "react-redux";
-
-import LibraryLayout from "readium-desktop/renderer/components/layout/LibraryLayout";
-
-import Header from "./Header";
-
 import { setLocale } from "readium-desktop/common/redux/actions/i18n";
-
-import { TranslatorProps, withTranslator } from "readium-desktop/renderer/components/utils/translator";
-
-import * as commonmark from "commonmark";
-
 import * as packageJson from "readium-desktop/package.json";
+import LibraryLayout from "readium-desktop/renderer/components/layout/LibraryLayout";
+import {
+    TranslatorProps, withTranslator,
+} from "readium-desktop/renderer/components/utils/translator";
+import * as info_de from "readium-desktop/resources/information/de.md";
+import * as info_en from "readium-desktop/resources/information/en.md";
+import * as info_fr from "readium-desktop/resources/information/fr.md";
```

Comment:
I think that using the compile-time WebPack loader for every single Markdown info file for each supported language is not a scalable solution. First of all, this source code will need to be updated when a locale is added. Secondly, the markdown files occupy memory unnecessarily.

Instead, I would suggest adding a compile-time WebPack directive to copy the info files into the app dist folder:
https://github.com/readium/readium-desktop/blob/6cf1e1b1f55b39a9a25aa77aac734e2418b8e3a0/webpack.config.main.js#L112-L129

Then, I would continue to use the following line of code:
```
let fileContent = await promisify(readFile)(`src/resources/information/${locale}.md`, {encoding: "utf8"});
```

...but I would use `__dirname` and `path.join()` to correctly reference `${locale}.md` in a cross-platform manner.

Reply:
__dirname returns '/' and not the same main process path

i'm looking for a solution but I don't see it

Reply:
https://medium.com/@dkerrious/electron-packager-file-path-issue-24f4bcbfc970

Reply:
For the renderer process, you can follow existing usage of `__dirname`:
https://github.com/readium/readium-desktop/blob/b2314a3947e534387183cf6bf37503d3874f5ce2/src/renderer/components/reader/Reader.tsx#L493-L509

Reply:
Note that in the particular case of "Thorium info Markdown files", you will not need `_NODE_MODULE_RELATIVE_URL`. But just for your information, this is used in the main process as well, so use the correct `node_module` location:

https://github.com/readium/readium-desktop/blob/541e48bbe45407a8918018e0076d4ca5ebecd68b/src/main/streamer.ts#L52-L58